qiskit/pulse/instructions/instruction.py

```diff
@@ -192,7 +193,7 @@ def shift(self: ScheduleComponent, time: int, name: Optional[str] = None) -> Sch
         """
         if name is None:
             name = self.name
-        return Schedule((time, self), name=name)
+        return Schedule((time, self), name=name, inplace=False)
```

Comment:
```suggestion
        return Schedule((time, self), name=name)
```
I'll let it raise once, because we don't want users to be able to follow this workflow and avoid getting the warning.